plugins/polls/app/controllers/polls_controller.rb

```diff
@@ -0,0 +1,94 @@
+class PollsController < ApplicationController
+  before_filter -> { require_plugin_enabled FoodsoftPolls }
+
+  def index
+    #@open_polls = Poll.open
+    @polls = Poll.page(params[:page]).per(@per_page)
+  end
+
+  def show
+    @poll = Poll.find(params[:id])
+  end
+
+  def new
+    @poll = Poll.new
+  end
+
+  def create
+    @poll = Poll.new(params[:poll])
+    @poll.created_by = current_user
+
+    if @poll.save
+      redirect_to @poll, notice: t('.notice')
+    else
+      render action: 'edit'
+    end
+  end
+
+  def edit
+    @poll = Poll.find(params[:id])
+
+    if @poll.created_by != current_user && !current_user.role_admin?
```

Comment:
What about adding a poll role? It's probably on your todo-list.
Not sure if we've already implemented additional roles for a plugin.

Reply:
we don't have this somewhere else ATM. it would require an additional col in the groups table, which i do not want to touch via a plugin. since i see no need for a new role, i'd like to avoid it for now

Reply:
:+1: